bulbs/styles/static/styles/bulbs-styles/components/video-series-detail.scss

```diff
@@ -0,0 +1,157 @@
+///////////////////////////////
+// ************************* //
+//    VIDEO SERIES DETAIL    //
+// ************************* //
+///////////////////////////////
+
+@import '../mixins/flexbox';
+@import '../mixins/flex-grid';
+
+.video-series-content {
+  padding: 0px;
+
+  .series-logo {
+    & img {
+      max-width: 100%;
+    }
+  }
+
+  ul.series-meta {
+    list-style-type: none;
+    padding: 10px;
+    text-align: center;
+
+    .series-seasons {
+      display: inline-block;
+      &:after {
+        content: '\2022';
+        font-style: normal;
+        margin: 0 .5em;
+      }
+    }
+    .series-episodes {
+      display: inline-block;
+    }
+  }
+
+  .latest-episode-data {
+    margin-bottom: 20px;
+  }
+
+  .video-series-ad {
+    border-top: solid 1px #eaeaea;
+    border-bottom: solid 1px #eaeaea;
```

Comment:
May merit making this color a variable since it's used all over the place in this file